Source/Plugins/Core/com.equella.core/js/tsrc/modules/LegacySelectionSessionModule.ts

```diff
@@ -215,6 +226,42 @@ export const buildSelectionSessionItemSummaryLink = (
   return itemSummaryPageLink;
 };
 
+/**
+ * Build a Selection Session specific Remote search Link. Recommended to first call `isSelectionSessionOpen()`
+ * before use.
+ * @param uuid The UUID of a Remote search
+ */
+export const buildSelectionSessionRemoteSearchLink = (uuid: string): string => {
+  const { stateId, integId } = getSelectionSessionInfo();
+  const remoteSearchLink = AppConfig.baseUrl.concat(
+    `access/z3950.do?.repository=${uuid}&uc=true&_sl.stateId=${stateId}`
+  );
```

Comment:
To keep things DRY so as to ensure accuracy, this should more so wrap the route in `routes`, something like:

```suggestion
  const remoteSearchLink = AppConfig.baseUrl.concat(
    `${routes.RemoteSearch.to(uuid)}&_sl.stateId=${stateId}`
  );
```

The same suggestion applies for the below Advanced Search.